### PR TITLE
Add patch for rancher-operator 0.2.0-alpha02

### PR DESCRIPTION
**Pull request by thedadams**
Issue:
https://github.com/rancher/rancher/issues/31954

### CODE REVIEW

File: packages/rancher-operator/package.yaml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 url: https://github.com/rancher/rancher-operator/releases/download/v0.2.0-alpha02/rancher-operator-0.2.0-alpha02.tgz
 packageVersion: 01
-releaseCandidateVersion: 01
+releaseCandidateVersion: 02
```
Comment (cbron) [inline packages/rancher-operator/package.yaml:3]: if packageVersion is 01 and rc version is 02, won't that make v0.2.0-alpha01-rc02 ? Should it be alpha02-rc01 ?

Comment (cbron) [reply packages/rancher-operator/package.yaml:3]: You can run `make charts` to check

Comment (thedadams) [reply packages/rancher-operator/package.yaml]: Yes. Fixed.

**Review by cbron — APPROVED** [review]
lgtm, but wait to merge until assets are removed

**Pull request merged**